cloud/shared/bin/lib/terraform.sh

```diff
@@ -32,6 +32,13 @@ function terraform::perform_apply() {
     )"
   fi
 
+  echo "Current working directory: ${PWD}"
+  if [[ -f "${TERRAFORM_TEMPLATE_DIR}/${TF_VAR_FILENAME}" ]]; then
+    echo "${TF_VAR_FILENAME} exists in ${TERRAFORM_TEMPLATE_DIR} directory"
+  else
+    echo "Cannot find ${TF_VAR_FILENAME} in ${TERRAFORM_TEMPLATE_DIR} directory"
```

Comment:
Should this exit if the file couldn't be found? Or is this a valid state? If it's valid, I'd consider putting what would happen in the echoed message (e.g. Cannot find XY in Z directory, falling back to defaults).

Reply:
probably yes-- i'm just mostly trying to debug quickly right now will iterate back when i find the error itself